pkg/controller/storagecluster/uninstall_reconciler_test.go

```diff
@@ -102,3 +104,30 @@ func assertCephClusterCleanupPolicy(
 	assert.Equal(t, CleanupPolicyConfirmation, cephCluster.Spec.CleanupPolicy.Confirmation)
 	assert.Equal(t, AllowUninstallWithVolumes, cephCluster.Spec.CleanupPolicy.AllowUninstallWithVolumes)
 }
+
+func TestDeleteStorageClasses(t *testing.T) {
```

Comment:
Use Table Driven approach and simplify the code a little bit.

Reply:
done